src/layouts/InterfaceLayout/components/InterfaceSideMenu/InterfaceSideMenu.vue

```diff
@@ -32,7 +32,7 @@
               aria-hidden="true"
```

Comment:
These tabs already open on initial load. Can you provide the issue number for this?